Sources/NIOTransportServices/NIOTSConnectionChannel.swift

```diff
@@ -157,7 +157,9 @@ internal final class NIOTSConnectionChannel: StateManagedNWConnectionChannel {
     internal var connectPromise: EventLoopPromise<Void>?
 
     internal var parameters: NWParameters {
-        NWParameters(tls: self.tlsOptions, tcp: self.tcpOptions)
+        let parameters = NWParameters(tls: self.tlsOptions, tcp: self.tcpOptions)
+        parameters.multipathServiceType = self.multipathServiceType
+        return parameters
```

Comment:
Is this change actually needed? I'd have thought it would work fine without it.

Reply:
I've inspected a bit more the code, and it seems that these changes aren't needed indeed